observability/src/main/java/io/grpc/observability/Observability.java

```diff
@@ -35,7 +38,8 @@ public static synchronized void grpcInit() {
     if (initialized) {
       throw new IllegalStateException("Observability already initialized!");
     }
-    LoggingChannelProvider.init(new InternalLoggingChannelInterceptor.FactoryImpl());
+    Sink sink = new GcpLogSink(PROJECT_ID);
```

Comment:
Pls add a TODO  comment saying this will be replaced with the configured destinationProjectId

Reply:
Also add a TODO saying once `Observability` is made into a singleton object, the close() on sink will be called from grpcFinish

Reply:
Done